File: exercicios/R/1178/1178.r
input = file('stdin','r')

a = as.integer(1)
c = as.double(0)
d = as.double(0)

n <- double(100)

b = as.double(readLines(input,n=1))
n[a] = b

for(a in 1:100){
    write(sprintf("N[%d] = %.4f",a-1,n[a]),'')
    b = b / 2
    n[a+1] = b
}

File: exercicios/JavaScript/1178/1178.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var v = parseFloat(lines.shift());

for(let i=0; i<100; i++){
    console.log("N["+ i + "] = " + v.toLocaleString('en-US', { minimumFractionDigits: 4 }));
    
    v /= 2;
}